### PR TITLE
Port tests in Postgres PL/R to PL/Container

### PR DESCRIPTION
**Pull request by xuebinsu**
## Description

This patch contains tests ported from Postgres PL/R testing UDFs written in R. Since R support is essential for PL/Container, it is good to have more tests that help to make PL/Container more stable. 

## Tests

No additional test is needed for this patch.

### CODE REVIEW

File: tests/sql/plr/opt_window.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  a == farg1[prownum]
+$$ window language plcontainer;
+
+select s, p, fast_win(NULLIF(s, 4), 123) over w
+from (
+  select s, s % 2 as p
+  from generate_series(1,10) s
+) foo
+window w as (partition by p order by s rows between unbounded preceding and unbounded following)
+order by s;
```
Comment (HaozhouWang) [inline tests/sql/plr/opt_window.sql]: We do not support window function yet

File: tests/sql/plr/do.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+do language plcontainer $$
+# container: plc_r_shared
+pg.throwlog("Hello, world!")
+$$;
+
+do language plcontainer $$
+# container: plc_r_shared
+pg.thrownotice("Hello, world!")
+$$;
```
Comment (HaozhouWang) [inline tests/sql/plr/do.sql]: in-line function does not support yet

File: tests/pl_schedule [hunk 3]
```diff
@@ -1,51 +1,58 @@
...
+#test: parallel_prepare
+#test: parallel_1 parallel_2 parallel_3 parallel_4 parallel_5 parallel_6 parallel_7 parallel_8
+#
+## PL/Container memory test
+#test: memory_consuming_python memory_consuming_r
+#test: memory_parallel_python memory_parallel_r
+#test: memory_parallel_python_1 memory_parallel_python_2 memory_parallel_python_3 memory_parallel_python_4 memory_parallel_python_5 memory_parallel_r_1 memory_parallel_r_2 memory_parallel_r_3 memory_parallel_r_4 memory_parallel_r_5
+#
+## PL/Container import pkg test
+#test: python_import_module r_import_library
```
Comment (HaozhouWang) [inline tests/pl_schedule]: seems many tests are commented out

Comment (xuebinsu) [reply tests/pl_schedule]: Fixed by uncommenting those lines.

**Pull request closed without merging**